packages/shell-api/bin/compile-shell-api.js

```diff
@@ -129,11 +136,12 @@ const loadAll = () => {
   const mainLib = yaml.load(main);
   const FILES = fs.readdirSync(yamlDir).filter((s) => (/[A-Z]/.test( s[0])));
   let exports = '\nexport default ShellApi;\n';
-  let types = [];
-  let typeConsts = [];
+  const types = [];
+  const typeConsts = [];
```

Comment:
not a review comment but a legit question: does typescript not complain that this is a const and you push into it? o.O o.O o.O

Reply:
For `let/const` the behaviour is like javascript: does not matter for `push` as we are not reassigning the variable. 

It would instead complain if the type is readonly:

```
const types: readonly string[] = []; // or Readonly<Array<string>>
types.push('something');
```